### ISSUE TITLE
the document of `url()` is wrong

### ISSUE THREAD

**Issue by l1t1** [quackscience/duckdb-extension-clickhouse-sql#17]
I copy the example from https://community-extensions.duckdb.org/extensions/chsql.html
```sql
D SELECT url('https://urleng.com/test','JSON');
Binder Error: Function "url" is a table function but it was used as a scalar function. This function has to be called in a FROM clause (similar to a table).
LINE 1: SELECT url('https://urleng.com/test','JSON');
               ^
```

**Comment by lmangani** [comment]
Thanks for the report! The documentation example needs to be fixed. URL is a TABLE macro.

**Comment by lmangani** [comment]
Fixed in chsql v1.0.6 for DuckDB v1.1.3
